plugins/org.locationtech.udig.project.ui/src/org/locationtech/udig/project/ui/internal/render/displayAdapter/impl/EventHandler.java

```diff
@@ -178,23 +188,23 @@ public void mouseMove( Event e ) {
     /**
      * @see org.eclipse.swt.events.MouseTrackListener#mouseEnter(org.eclipse.swt.events.MouseEvent)
      */
-    public void mouseEnter( Event e ) {
+    public void mouseEnter(Event e) {
```

Comment:
Just wondering why `@Override` was added in other classes of this changes but not in this one. Please remove `@see` javadocs as well